css/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  /* margin: 0 auto; */
+  font-family: 'Inter', sans-serif;
+  background: #fff;
+}
+
+header {
+  display: flex;
+  height: 48px;
+  align-items: baseline;
+  padding: 8px 24px;
+  color: #28352f;
+}
+
+h1.logo {
+  flex: 1;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+span.menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.menu .bar {
+  border: 2px solid black;
+  padding: 3px;
+  width: 20px;
+  border-bottom: none;
+  border-left: none;
+  border-right: none;
+}
+
+section.headline {
+  height: 100px;
+  background: url("../img/header.png") center 0% no-repeat;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+.intro {
+  font-size: 40px;
+  font-weight: 700;
```

Comment:
- [ ] Please change the `font-family` to the same font used in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design.

Reply:
I already declared that in the body style so I think there is no need to do this again because the font clearly affects this portion of the code you highlighted:

`body {
  font-family: 'Inter' sans-serif;
}`

Please let me know if you think otherwise.

Reply:
Hello! @Code-panther font-family used in the Figma design is `Poppins`.
![Captura de pantalla_2021-09-16_12-16-14](https://user-images.githubusercontent.com/53111464/133656321-6b3b4244-f989-4033-b5d7-8669a73a085e.png)

Reply:
The intro section is completed!

Reply:
Thank you!